.travis.yml

```diff
@@ -17,12 +17,29 @@ script:
 after_success:
 - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage
 
+before_deploy: 
+  - npm run build
+  - cd lib/
```

Comment:
Why changing directory to lib/ ?
If you are attempting to modify what is deployed as part of the package we can use `files` section of the `package.json`

Reply:
ex) https://github.com/microsoft/vott-react/blob/master/package.json

Reply:
oh I wasn't aware you could do that, that's pretty cool